src/components/pages/Categories/List.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="categories-list">
+    <categoryPost class="post-width" />
+    <categoryList
+      :categories="categories"
+      :theads="theads"
+      class="list-width"
+    />
+  </div>
+</template>
+
+<script>
+import categoryList from '@Components/molecules/CategoryList/index.vue';
+import categoryPost from '@Components/molecules/CategoryPost/index.vue';
+
+export default {
+  components: {
+    categoryList,
+    categoryPost,
+  },
```

Comment:
[nits]

細かいですが、他のファイルでは以下のようになっていると思うので、修正をお願いします。

```suggestion
import CategoryList from '@Components/molecules/CategoryList/index.vue';
import CategoryPost from '@Components/molecules/CategoryPost/index.vue';

export default {
  components: {
    categoryList: CategoryList,
    categoryPost: CategoryPost,
  },
```

Reply:
承知いたしました！ご指摘いただきありがとうございます